jerry-core/api/jerry.c

```diff
@@ -254,6 +255,44 @@ jerry_gc (void)
   ecma_gc_run (JMEM_FREE_UNUSED_MEMORY_SEVERITY_LOW);
 } /* jerry_gc */
 
+#ifdef JMEM_STATS
+void jmem_heap_get_stats (jmem_heap_stats_t *);
+#endif
+/**
+ * Get heap memory stats.
+ *
+ * @return true - get the heap stats successful
+ *         false - otherwise. Usually it is because the MEM_STATS feature is not enabled.
+ */
+bool
+jerry_get_memory_stats (jerry_heap_stats_t *out_stats_p) /**< [out] heap memory stats */
+{
+#ifdef JMEM_STATS
+  if (out_stats_p == NULL)
+  {
+    return false;
+  }
+
+  jmem_heap_stats_t jmem_heap_stats = {0};
+  jmem_heap_get_stats (&jmem_heap_stats);
+
```

Comment:
Ok, `if (jerry_get_memory_stats(VERSION, &stats))` the VERSION here should be sth like `JERRY_API_MAJOR_VERSION`. I still feel strange because it is the only API who pass the api version. 
e.g.
`jerry_get_own_property_descriptor` is also getting a block of memory as jerry_property_descriptor_t, but it doesnt pass the api version.